include/msgpack/adaptor/cpp11/unordered_map.hpp

```diff
@@ -42,9 +42,9 @@ struct convert<std::unordered_map<K, V>> {
         for(; p != pend; ++p) {
             K key;
             p->key.convert(key);
-            p->val.convert(tmp[key]);
+            p->val.convert(tmp[std::move(key)]);
         }
-        tmp.swap(v);
+        v = std::move(v);
```

Comment:
Thank you for fixing. I'm on a business trip now. I will merge the PR next week.